server/app.js

```diff
@@ -15,6 +15,77 @@ app.engine(
   })
 );
 
+let pg = require("pg");
+
+var knex = require("knex")({
+  client: "pg",
+  connection: {
+    host: "127.0.0.1",
+    user: "cyf",
+    password: "password",
+    database: "orula",
+    port: 5432
+  }
+});
+
+// get '/classes'
+app.get("/classes", (req, res) => {
+  const sqlStatement = "select * from classes";
+  knex.raw(sqlStatement).then(data => {
+    res.json(data);
+  });
+});
+const callBack = (error, classes) => {
+  if (error) {
+    console.error(error);
+    res.sendStatus(500);
+  } else {
+    res.json(classes);
+  }
+};
+// get '/classes/class_id/:id'
+app.get("/classes/class_id/:id", (req, res) => {
+  const sqlStatement = `select * from classes where class_id =${req.params.id}`;
+  knex.raw(sqlStatement).then(data => {
+    res.json(data);
+  });
+});
+// post '/classes/'
+app.post("/classes", function(req, res) {
+  const body = req.body;
+  const sqlStatement = `INSERT INTO classes
```

Comment:
similar to what we did in api helper in client, could you separate DB calls in a helper file  - Look at this for examples https://github.com/CodeYourFuture/edesia/blob/master/server/helpers/db.js